docs/documentation/server_admin/topics/admin-console-permissions/fine-grain-v2.adoc

```diff
@@ -66,17 +66,17 @@ set of management operations:
 
 [NOTE]
 ====
-- An administrator must have explicit permission assigned for each operation to perform the corresponding action. For example, 
-  assigning only *manage* without *view* will prevent the user from being visible.  
-- The *view* operation also considers group-related permissions, specifically *view-members* for viewing group members. 
-  The evaluation takes permissions for specific resources (specific user permissions and specific group permissions) first. 
+- An administrator must have explicit permission assigned for each operation to perform the corresponding action. For example,
+  assigning only *manage* without *view* will prevent the user from being visible.
+- The *view* operation also considers group-related permissions, specifically *view-members* for viewing group members.
+  The evaluation takes permissions for specific resources (specific user permissions and specific group permissions) first.
   See <<_resolving-conflicting-permissions, Resolving Conflicting Permissions>> for more information.
-- Similarly, the *manage* operation takes *manage-members* into account when managing group members.  
+- Similarly, the *manage* operation takes *manage-members* into account when managing group members.
```

Comment:
@pedroigor we should probably mention here that this "taking-into-account" applies to impersonation as well, wdyt?

Reply:
Added a second commit with some updates to the docs. I can send a separate PR if you prefer, but perhaps we can just agree on it here.